image/spec/README.md

```diff
@@ -23,6 +23,15 @@ and in [OCI Image Format Specification](https://github.com/opencontainers/image-
 
 All 1.X versions are compatible with older ones.
 
+### [v1.3](https://github.com/moby/moby/blob/ff46faa238f7ea7e8deddfde6c46cdde07fd680c/image/spec/v1.3.md)
+
+* Implemented in Docker v25.0 (to be released)
```

Comment:
It's fine to remove the `(to be released)` here; I'm even considering it we need to continue including Docker versions